theme/scripts/cart.ts

```diff
@@ -216,7 +216,7 @@ export default(function() {
       }
```

Comment:
For lines 210-216 (github won't actually let me comment on lines that are collapsed in the PR review zone)

These two calls might be able to be simplified:

from:
```
  const cartHtml = await Cart.getCartHtml();
  Cart.render(cartHtml);
  
  const cartItemCount = document.querySelector(Cart.SELECTORS.cartItemCount).getAttribute(Cart.ATTRIBUTES.itemCount);
  if (cartItemCount) {
    Cart.updateCartItemCount(cartItemCount);
  }
```
to simply:

```
await Cart.render()
Cart.updateCartItemCount()
```

Then let these two functions deal with getting the `cartHtml` and the `cartItemCount`, respectively